Makefile.conf.template

```diff
@@ -81,20 +88,20 @@ DEFS+= -DDBG_MALLOC #Include additional, debug-enabled allocator flavors
 #DEFS_GROUP_START
 #DEFS+= -DFAST_LOCK #Use fast architecture specific locking
 #DEFS+= -DUSE_SYSV_SEM #Use SYSV sems for locking ( slower & limited number of locks
-#DEFS+= -DUSE_PTHREAD_MUTEX #Use pthread mutexes for locking
+DEFS+= -DUSE_PTHREAD_MUTEX #Use pthread mutexes for locking
 #DEFS+= -DUSE_UMUTEX #Use FreeBSD-specific low-level mutexes for locking
 #DEFS+= -DUSE_POSIX_SEM #Use POSIX sems for locking
 #DEFS_GROUP_END
-#DEFS+= -DUSE_FUTEX #Use linux futexes with fast architecture specific locking
+DEFS+= -DUSE_FUTEX #Use linux futexes with fast architecture specific locking
 #DEFS+= -DBUSY_WAIT #Use busy waiting on the lock
 #DEFS+= -DDBG_LOCK #Attach debug info to all lock structures
 #DEFS+= -DDBG_STRUCT_HIST #Include support for struct history logging
 #DEFS+= -DDBG_TCPCON #Attach struct history info to all TCP connections
 #DEFS+= -DNOSMP #Do not use SMP compliant locking. Faster but won't work on SMP machines
 #DEFS+= -DEXTRA_DEBUG #Compile in some extra debugging code
-DEFS+= -DCC_O0 #Zero compiler optimizations (FAST compile, SLOW code. For devs)
+#DEFS+= -DCC_O0 #Zero compiler optimizations (FAST compile, SLOW code. For devs)
 #DEFS+= -DORACLE_USRLOC #Use Oracle compatible queries for USRLOC
 #DEFS+= -DSHM_EXTRA_STATS #Provide tools to get extra statistics for the shared memory used
 #DEFS+= -DUNIT_TESTS #Include unit testing code into opensips and modules
 
-PREFIX ?= /usr/local/
+PREFIX ?= /usr/
```

Comment:
Why change this here? This should be overwritten in the `PKGBUILD`.

Reply:
Moreover, the flags are configured of the `master` branch, which should be used **only** for development.

Reply:
You are right, PREFIX change can be omitted. 

What flags should be used instead if targeting a production environment??